ime/app/src/main/java/com/anysoftkeyboard/dictionaries/SuggestionsProvider.java

```diff
@@ -264,60 +279,74 @@ public void setupSuggestionsForKeyboard(
         mCurrentSetupHashCode = newSetupHashCode;
         final CompositeDisposable disposablesHolder = mDictionaryDisposables;
 
-        for (DictionaryAddOnAndBuilder dictionaryBuilder : dictionaryBuilders) {
+        for (int i = 0; i < dictionaryBuilders.size(); i++) {
```

Comment:
I would add `DictionaryAddOnAndBuilder dictionaryBuilder = dictionaryBuilders.get(i);` and use `dictionaryBuilder` everywhere you have `dictionaryBuilders.get(i)`

Reply:
Thanks! This makes the code much simpler ;)